lib/src/date_page_view.dart

```diff
@@ -52,19 +51,56 @@ class _DatePageViewState extends State<DatePageView> {
           axisDirection: AxisDirection.right,
           offset: position,
           anchor: 0,
-          slivers: <Widget>[
-            SliverFillViewport(
-              viewportFraction: 1 / visibleDays,
-              delegate: SliverChildBuilderDelegate(
-                (context, index) => widget.builder(
-                  context,
-                  LocalDate.fromEpochDay(index + visibleDays ~/ 2),
-                ),
-              ),
-            ),
-          ],
+          center: _centerKey,
+          slivers: _slivers,
```

Comment:
What's the benefit of two separate slivers? Couldn't we just adjust the offset and limit of the existing sliver?

Reply:
the benefit is that it is actually infinite in both directions. Also, it simplifies dates calculation since the initial date becomes a center date in that case.

An additional plus with this approach is that - range can be changed dynamically, that potentially should simplify animation feature implementation (I saw it in your todo list). Count values are changeable and which means that it will be easier to update min/max values without the need to recalculate offset of the visible date. And overall calculation on scroll become simpler - you can notice in that I basically kept 1 implementation of offset calculation for both Days and Week ranges